components/pagination/Pagination.js

```diff
@@ -0,0 +1,128 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import { LinkButton } from '../button';
+import { TextSmall } from '../typography';
+import { IconChevronLeftMediumOutline, IconChevronRightMediumOutline } from '@teamleader/ui-icons';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class Pagination extends PureComponent {
+  static propTypes = {
+    inverse: PropTypes.bool,
+    nextPageText: PropTypes.string,
+    numItems: PropTypes.number.isRequired,
+    numItemsPerPage: PropTypes.number,
+    maxNumPagesVisible: PropTypes.number,
+    currentPage: PropTypes.number.isRequired,
+    onPageChange: PropTypes.func.isRequired,
+    prevPageText: PropTypes.string,
+    className: PropTypes.string,
+  };
+
+  static defaultProps = {
+    currentPage: 1,
+    inverse: false,
+    numItemsPerPage: 10,
+    maxNumPagesVisible: 7,
+  };
+
+  render() {
+    const {
+      className,
+      currentPage,
+      inverse,
+      maxNumPagesVisible,
+      nextPageText,
+      numItems,
+      numItemsPerPage,
+      onPageChange,
```

Comment:
Does it make sense to treat this component more like an `input`, and rename this from `onPageChange` just to `onChange`? The component is called `Pagination`, so `page` is implied. WDYT? 🤔